finality-aleph/src/party/manager/aggregator.rs

```diff
@@ -209,6 +208,13 @@ where
                 exit,
             )
             .await;
+            let result = match result {
+                Ok(_) => Ok(()),
+                Err(err) => {
```

Comment:
If you have a real error then better return it rather than masking it here.

Reply:
Do you have something more specific on your mind, cuz I can't just return some error here without refactoring Task/Subtask, etc.

Reply:
Ugh, they probably need a refactor then. Don't necessarily do this in this PR, but please create a task for it.

Reply:
done (added task)